frontend/src/css/Sidebar.styl

```diff
@@ -116,22 +128,19 @@ leftside()
             text-decoration underline
             color fourthColor
 
-/***** MEDIA 992px and 1199px ******/
 
-@media (min-width: min-lg) and (max-width: max-lg)
+/***** MEDIA 576px and 1199px ******/
 
-  .sidebar
-    position fixed
-
-/***** MEDIA 576px and 991px ******/
+// Regroup the css rules in one mediaqueries
 
-@media (min-width: min-sm) and (max-width: max-md)
+@media (min-width: min-sm) and (max-width: max-lg)
```

Comment:
this is forbiden. Do not bypass mediaqueries.
if you want to apply the sames rules from min-sm to min-md AND from min-md to min-lg AND in lg, duplicates the rules in every "sub"mediaqueries